### ISSUE TITLE
Goodtwitter2 not showing up after I open a new window, works only after a refresh

### ISSUE THREAD

**Issue by MrCelroy** [Bl4Cc4t/GoodTwitter2#91]
**Describe the bug**
Goodtwitter2 not showing up after I open a new window, shows up only after a refresh
This happens every time I open a new window even with twitter running in another tab
**Video**
https://www.youtube.com/watch?v=4Zx3OZMS95A

**Your configuation**
 - Chrome Version 83.0.4103.97
 - Tampermonkey v4.10

**Comment by MrCelroy** [comment]
Alright so I discovered that it only occurs when I go directly to 
twitter.com
when I go to twitter.com/home it works without refreshing

**Comment by Bl4Cc4t** [comment]
ah yeah, I forgot to check if the user is logged out on `/`.

**Comment by MrCelroy** [comment]
so when's this fix coming out?

**Comment by Bl4Cc4t** [comment]
Hm, I initially wanted to improve the disableAutoRefresh feature for the next version but I can’t get it to work...
I guess I have to postpone that :/
So I guess later today will be the release of v0.0.21 :)
